docs/requirements-and-installation.md

```diff
@@ -32,6 +32,9 @@ The ANTA package and the cli require some packages that are not part of the Pyth
 pip install anta
 ```
 
+!!! warning
+    AVD and ANTA version requirements are currently strict. Check ANTA's page on AVD's `eos_validate` Role for details.
+
```

Comment:
Hmm maybe we should add a link to the page and I don't know if it should be at the top of the page or rather a section at the end.
ANTA is meant to live without AVD as well.

thoughts @titom73 ?

```suggestion
!!! warning
    When using ANTA mode in [AVD](https://avd.arista.com) `eos_validate` role, (currently in preview), ensure you install the documented supported ANTA version for your AVD version. The latest documented version can be found at: https://avd.arista.com/stable/roles/eos_validate_state/ANTA-Preview.html

```